dsls/sql/src/test/java/org/apache/beam/dsls/sql/planner/MockedBeamSqlTable.java

```diff
@@ -49,6 +48,7 @@
   public static final ConcurrentLinkedQueue<BeamSqlRow> CONTENT = new ConcurrentLinkedQueue<>();
 
   private List<BeamSqlRow> inputRecords;
+  private PCollection.IsBounded isBounded = PCollection.IsBounded.BOUNDED;
```

Comment:
The core feature of `TestStream` is enable tests to advance the watermark and processing time as elements are emitted. Which seems not needed for our test? We just need `IsBounded` to tell whether specified tables can join?